package.json

```diff
@@ -75,6 +75,7 @@
     "cucumberjs-allure-reporter": "1.0.3",
     "expect": "^23.0.0",
     "express": "^4.16.2",
+    "file-saver": "^2.0.2",
```

Comment:
is it required for pay-web-component?

Reply:
Yep, the latest version requires both `file-saver` and `xlsx` - fails to compile without.

Reply:
Isnt it an overhead for us to maintain those standalone dependencies versions? @danlysiak @smathangi

Reply:
Agree @hemantt  ideally those should part of pay web-component dependency itself.

Reply:
👍 I was advised yesterday that there were issues trying to incorporate into the component itself, but requested they look to sort in future versions so we no longer need to maintain them.